static/app/components/group/streamlinedParticipantList.tsx

```diff
@@ -0,0 +1,113 @@
+import {useTheme} from '@emotion/react';
+import styled from '@emotion/styled';
+import {FocusScope} from '@react-aria/focus';
+
+import Avatar from 'sentry/components/avatar';
+import AvatarList from 'sentry/components/avatar/avatarList';
+import TeamAvatar from 'sentry/components/avatar/teamAvatar';
+import {Overlay, PositionWrapper} from 'sentry/components/overlay';
+import {t, tn} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {Team, User} from 'sentry/types';
+import useOverlay from 'sentry/utils/useOverlay';
+
+interface DropdownListProps {
+  users: User[];
+  teams?: Team[];
+}
+
+export default function ParticipantList({users, teams}: DropdownListProps) {
+  const {overlayProps, isOpen, triggerProps} = useOverlay({
+    position: 'bottom-start',
```

Comment:
I'd add `isKeyboardDismissable: true` and `shouldCloseOnBlur: true` here, it seems a bit too sticky as is